composer.json

```diff
@@ -36,11 +36,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Yiisoft\\Db\\Pgsql\\": "src"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
+            "Yiisoft\\Db\\Pgsql\\": "src",
             "Yiisoft\\Db\\Pgsql\\Tests\\": "tests"
```

Comment:
That does not look correct. There is no need to load tests when package is used in the application.

Reply:
I know what autoload and autoload-dev means, currently db loads the tests in autoload, I am investing it to be able to make everything independent and in the end place all the tests in autoload-dev with final class .